jupyterlab_gcpscheduler/src/components/dialog.tsx

```diff
@@ -164,10 +167,9 @@ export class SchedulerDialog extends React.Component<Props, State> {
   }
 
   render() {
-    const { gcpSettings } = this.state;
-    const projectId = gcpSettings ? gcpSettings.projectId : '';
+    const projectId = this.getProjectId();
     return (
-      <Dialog open={this._isOpen()}>
+      <Dialog open={this._isOpen()} scroll="body">
```

Comment:
The difference is best illustrated here https://material-ui.com/components/dialogs/#scrolling-long-content

Reply:
I see. I actually thought it might be the "paper" behavior where only the body of the dialog scrolls but the overall document that it appears in doesn't. How does this actually look when you're submitting a job?

Reply:
http://go/scrcast/NTc3ODgwNDkwMTQxMjg2NHwzNjBhN2RmOC0zZQ

I think it looks a lot cleaner than the paper, which make the form seem cramped.

Reply:
I agree. Let's roll with it for now and if UX has thoughts otherwise, we can pivot later. I'm starting to wonder if our form is going to be too big to justify a dialog and instead require a new tab.